zbsprimarystorage/zbsprimarystorage/zbsutils.py

```
@@ -55,11 +89,34 @@ def query_volume_info(logical_pool, volume):
     return shell.call("%s query file --path %s/%s --format json" % (ZBS_BIN_PATH, logical_pool, volume))
 
 
+def query_volumes_in_logical_pool(logical_pool_name):
+    return shell.call("%s list file --pool %s --format json" % (DEFAULT_ZBS_BIN_PATH, logical_pool_name))
```

Ruff (0.14.4)

`shell` may be undefined, or defined from star imports

(F405)